### PR TITLE
FEAT: cart page api integration

### PR DESCRIPTION
**Pull request by ArjunGTX**
- integrated cart page api

### CODE REVIEW

File: src/pages/cart/Cart.jsx [hunk 1]
```diff
@@ -1,48 +1,44 @@
...
-    image: "assets/images/helmets/racing/agv/pista-gp-rr-min.png",
-  },
-];
-
 export const Cart = () => {
-  const navigate = useNavigate();
-  const { auth } = useAuth();
-  useEffect(() => !auth.isLoggedIn && navigate("/login"), []);
+  const { cart } = useCart();
+  console.log(cart);
```
Comment (prathmeshjagtap) [inline src/pages/cart/Cart.jsx]: you can remove console.log

**Review by tanishk15sharma — APPROVED** [review]
I liked you are doing axios only in one file . other things like naming conventions are destructuring is also good .

**Review by prathmeshjagtap — APPROVED** [review]
Code is Easily Readable, I liked that you have made different  functions for each task

File: src/pages/cart/Cart.jsx (hunk 1 above)
Comment (ArjunGTX) [reply src/pages/cart/Cart.jsx]: Yeah, I will. Old debugging habbits!

**Pull request merged**